dev/NavigationView/NavigationView.h

```diff
@@ -165,7 +165,6 @@ class NavigationView :
     void UpdateVisualState(bool useTransitions = false);
     void UpdateVisualStateForOverflowButton();
     void UpdateLeftNavigationOnlyVisualState(bool useTransitions);
-    void UpdateNavigationViewUseSystemVisual();
     static void PropagateShowFocusVisualToAllNavigationViewItemsInRepeater(winrt::ItemsRepeater const& ir, bool showFocusVisual);
```

Comment:
>PropagateShowFocusVisualToAllNavigationViewItemsInRepeater [](start = 16, length = 58)

I think this isn't called anymore?

Reply:
Yes, this isn't used anymore, missed that function. Is removed now.